functionary/ui/templates/base.html

```diff
@@ -86,6 +87,14 @@ <h1 class="title is-1">
                             </a>
                         </li>
                     </ul>
+                    <ul class="menu-list block">
```

Comment:
I vote to put this below "Functions" and to call it "Schedules" for the sake of consistency.

Also, the menu highlighting doesn't work because the javascript that drives that is keying off of `-list` being in the url name (i.e. schedule-list).  That might be a overly restrictive, but also having the url name be consistent with the others would probably be a good thing.

Reply:
Fixed. Also shortened the url names for scheduled tasks